ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,50 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Conv2D",
+    x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC", "NCHW"]),
+        padding=st.sampled_from(["SAME", "VALID"]),
+        type="2d",
+        dilation_min=1,
+        dilation_max=1,
+    ),
+    test_with_out=st.just(False),
+    number_positional_args=st.just(0),
```

Comment:
I think you should remove `number_positional_args`. I does not need to be explicitly set.

Reply:
ok